media-libs/openexr/openexr-3.2.3-r1.ebuild

```diff
@@ -29,7 +29,7 @@ REQUIRED_USE="doc? ( utils )"
 RESTRICT="!test? ( test )"
 
 RDEPEND="
-	app-arch/libdeflate[zlib]
+	app-arch/libdeflate:=
```

Comment:
These need to use `zlib(+)` instead in order to handle the case where the user didn't update libdeflate, but did update openexr.

Reply:
If the user doesn't update libdeflate, he has app-arch/libdeflate[zlib] installed and nothing changes.

`zlib(+)` would be required if we'd left the old versions in the tree that had `[zlib]` and we'd have to enable `zlib` in the case the flag exists. But there are no versions where that is the case.

Reply:
That isn't right. You need to cater for installed packages (i.e. the VDB). That's the main point of (+) and (-).

Reply:
Yeah I'm a bit baffled by that response too (and don't understand why it was marked as resolved).

Reply:
I went ahead and added `[zlib(+)]` where appropriate in c5552fd1faac0ba4997cc0ed08fcc4d4930800fd.